src/translations/pt-BR.json

```diff
@@ -16,6 +16,16 @@
             "description": "Explore curtas, clipes, e novos formatos que nossa comunidade inventa todos os dias."
         }
     },
+    "myVideos": {
+        "title": "My videos",
+        "item": {
+            "status": {
+                "label": "Status",
+                "published": "Publicado",
+                "unpublished": "Não publicado"
+            }
+        }
+    },
```

Comment:
(I don't speak Portuguese but) are all of these actually translated?

I think we should leave files other than `en.json` untouched so that they can be fully translated later by Daniel

Reply:
Ok. If it is fine to Daniel take care of that I'll not replicate the keys